### PR TITLE
ENH patch for osx-arm64 code signing

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
I think the patchs conflict for path_actions.py

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,2 +1,4 @@
-provider: {linux_aarch64: default, linux_ppc64le: default}
+build_platform: {osx_arm64: osx_64}
```
Comment (isuruf) [inline conda-forge.yml:1]: We still can't build this on arm64 because pyinstaller doesn't support python 3.8

Comment (beckermr) [reply conda-forge.yml:1]: Ahhhh ok. I will remove that bit.

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: conda-forge.yml (hunk 1 above)
Comment (nehaljwani) [reply conda-forge.yml]: xref: https://github.com/pyinstaller/pyinstaller/issues/4535#issuecomment-703067625

**Comment by beckermr** [comment]
@conda-forge/conda-standalone this is ready for review!

**Review by isuruf — APPROVED** [review]

**Pull request merged**